src/SFA.DAS.Forecasting.Domain.UnitTests/Estimations/AccountEstimationTests.cs

```diff
@@ -144,5 +147,17 @@ public void Add_Virtual_Apprenticeship_Assigns_FundingSource_To_Apprenticeship()
             var apprenticeship2 = estimation.AddVirtualApprenticeship("course-1", "test course", 1, 1, 2019, 5, 18, 1000, FundingSource.Levy);
             Assert.AreEqual(apprenticeship2.FundingSource, FundingSource.Levy);
         }
+
+        
+        [TestCase(-20, 12, false)]
+        [TestCase(-15, 12, false)]
+        [TestCase(-5, 10, true)]
+        public void Has_Valid_Apprenticeships_Returns_False_When_No_Active_Apprenticeshp(int deductNumberOfMonths, int numberOfMonths, bool isValid)
+        {
+            var estimation = ResolveEstimation();
+            var startDate = DateTime.Now.AddMonths(deductNumberOfMonths);
```

Comment:
change the  variable name deductNumberOfMonths

Reply:
changed